.changeset/fix-externals-array-validation.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/deployer': patch
+---
+
+Fixed `mastra build` failing on a package you already externalized, without breaking builds that use externalized packages while they load.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)